products/firefox_iOS/testing_iOS.md

```diff
@@ -0,0 +1,22 @@
+# Testing on iOS for a new release
+We currently rely on screenshots for testing. These are provided on the [dev.l10n mailing list](https://lists.mozilla.org/listinfo/dev-l10n), which you should follow if you are working on l10n for any of the existing Mozilla products (otherwise you are missing crucial information and updates).
+
+## Testing with screenshots
+Screenshots are currently provided by the iOS team on dev.l10n, and the usual link where you can check out the latest versions can be found [right here](https://people-mozilla.org/~sarentz/fxios/screenshots/).
```

Comment:
Nitpicking: it's not the link that can be found at the link.

`… on dev.l10n and they’re available in [this page](https://people-mozilla.org/~sarentz/fxios/screenshots/).`